components/UserProduct.vue

```diff
@@ -8,7 +8,7 @@
         <h2>{{ product.title }}</h2>
         <p class="pt-3 ma-0">{{ product.text }}</p>
         <v-btn class="action-btn px-5" :depressed="true">
-          Open
+          <span>Open</span>
```

Comment:
why the span here?

Reply:
1. Consistently with other, similar elements. (That's how I found this one.)
2. Best practice, yes. It allows us to uniquely select the text for styling (CSS) or testing (can use `toBe()` instead of `toContain()`).

Reply:
okay yeah that makes sense, thanks